microsoft-edge/devtools-guide-chromium/whats-new/2021/07/devtools.md

```diff
@@ -0,0 +1,288 @@
+
+<!-- todo: comment-out or delete steps? -->
+1. In Microsoft Edge Canary, navigate to [Microsoft ToDo](https://to-do.office.com/tasks/inbox).
+1. Open DevTools, then select the **Settings** (gear) icon.
+1. Navigate to the **Experiments** section, then select the **Detached Elements** checkbox.
+1. Select **X** to close **Settings**.  Select `F12` twice to restart DevTools.
+1. Select the **+** (**More Tools**) button and then select **Detached Elements**.
+1. In the webpage, select **Get started**.  If prompted, select a user account to login.
+1. Select the **+** next to **Add a task**, and then select **Add**.
+1. Select the task you created, select **Add due date**, select **Pick a date**, and then select a date from the calendar.  Then select **Save**.
+1. In DevTools, in the **Detached Elements** tool, select **Get Detached Elements**.
+1. If the **Analyze** button is grayed-out, add another task and date, then select **Get Detached Elements** again.<!--added extra step-->
+1. Select the **Analyze** button in the toolbar and wait for the **Id** column to populate.
+1. In the **Id** column, select an ID.  The **Memory** tool opens in the bottom panel, and displays the retainers list.
+
+:::image type="complex" source="../../media/2021/07/detached-elements-tool.msft.png" alt-text="The Detached Elements tool" lightbox="../../media/2021/07/detached-elements-tool.msft.png":::
+   The **Detached Elements** tool
+:::image-end:::
+
+For more information, navigate to the **Detached Elements** section in [Experimental features](../../../experimental-features/index.md).
+<!-- todo: link to subheading in page, when available -->
+
+<!-- no crbug -->
+
+
+<!-- 3 -->
+## Microsoft Edge Developer Tools for Visual Studio Code now integrates with the Visual Studio Code Debugger workflow
+
+<!-- Title: While debugging, launch the DevTools extension by selecting the Inspect button -->
+<!-- Subtitle: Microsoft Edge DevTools for Visual Studio Code now integrates seamlessly with the JavaScript debugging workflow in the editor. -->
+
+If you use the JavaScript debugging workflow in Visual Studio Code, you can now launch the DevTools extension by selecting the **Inspect** button.
+
+:::image type="complex" source="../../media/2021/07/inspect-button.msft.png" alt-text="The Inspect button in Visual Studio Code to launch the DevTools extension" lightbox="../../media/2021/07/inspect-button.msft.png":::
+   The **Inspect** button in Visual Studio Code to launch the DevTools extension
+:::image-end:::
+
+This feature integrates DOM and CSS debugging with the JavaScript debugging workflow in Visual Studio Code.  If you don't have the DevTools extension installed, when you select the **Inspect** button, Visual Studio Code prompts you to install the extension seamlessly.
+
+Other new features are:
```

Comment:
I feel like the order of this list should be reversed. So it should be:

- The tools refresh automatically when you switch between different debugging targets.
- Several bug fixes.
- More detailed documentation of the extension.